packages/xrpc-server/src/types.ts

```diff
@@ -238,20 +277,52 @@ export function isHandlerError(v: unknown): v is HandlerError {
 }
 
 export class InvalidRequestError extends XRPCError {
-  constructor(errorMessage?: string, customErrorName?: string) {
-    super(ResponseType.InvalidRequest, errorMessage, customErrorName)
+  constructor(
+    errorMessage?: string,
+    customErrorName?: string,
+    options?: ErrorOptions,
+  ) {
+    super(ResponseType.InvalidRequest, errorMessage, customErrorName, options)
+  }
+
+  [Symbol.hasInstance](instance: unknown): boolean {
```

Comment:
Allows `err instanceof InvalidRequestError` to return `true` when error was build using `new XRPCError(ResponseType.InvalidRequest, ...)`